docs/index.rst

```diff
@@ -3,12 +3,9 @@ Welcome to Dockstore Documentation!
 
 .. note:: Our code lives on GitHub at `dockstore/dockstore <https://github.com/dockstore/dockstore>`_ and `dockstore/dockstore-ui2 <https://github.com/dockstore/dockstore-ui2>`_.
 
-Dockstore is an open platform used by the `GA4GH <https://www.ga4gh.org/>`__ for sharing Docker-based tools, workflows, and services.
-For tools and workflows, we support Common Workflow Language (CWL), Workflow Description Language (WDL), Nextflow (NFL), and Galaxy.
+Dockstore is an open platform used by the `GA4GH <https://www.ga4gh.org/>`__ for sharing Docker-based tools, workflows, and services. For tools and workflows, we support Common Workflow Language (CWL), Workflow Description Language (WDL), Nextflow (NFL), and Galaxy.
```

Comment:
Galaxy isn't docker-based... maybe instead said "used by GA4GH for sharing reproducible and interoperable tools, workflows, and services."